_local/platform/darwin.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+### BEGIN Platform Specific Section
+
+function preDockerCommands() {
+    local tag=$1
+    # Docker Desktop for Darwin is stupid, and doesn't read FROM image
+    # Also fails with confusing "denied: request access" in addition to error
+    # when build fails, instead of just error.
+    if [ "$(docker image ls -q --filter "reference=golang:$tag")" == "" ]; then
+        docker image pull "golang:$tag"
+    fi
+}
+
+### END Platform Specific Section
+
+VERSION=$(bash --version | grep version)
```

Comment:
Not sure why we need to enforce a bash version if we would run all the tests within a container we control. Do you mind clarifying why this is needed please?

Reply:
The scripts are running the containers, so this version of bash is mainly for Darwin. The built-in bash shell distributed with Darwin is an older 3.x one, and having written a lot of scripts for managing Mac users envs in IT related activities, I've found it simpler to at least upgrade to bash 4 or 5. But if you'd rather, I'll remove the requirement.